### ISSUE TITLE
Cannot use derivative function

### ISSUE THREAD

**Issue by aftnix** [influxdata/influxdb#7110]
```
> select "bytes_recv" from "net" where "host" = 'wowza1.ca.iptv' limit 10;
name: net
---------
time			bytes_recv
1469856950000000000	42522820906163
1469856950000000000	613799672816
1469857000000000000	42523169448160
1469857000000000000	613799745660
1469857050000000000	42523518763714
```

```
> select derivative("bytes_recv") from "net" where "host" = 'wowza1.ca.iptv' and time > now() - 5m;
ERR: invalid character 'j' looking for beginning of value
```

```
[aftnix@localhost ~]$ influx -version
InfluxDB shell version: 0.13.0
```

**Comment by jsternberg** [comment]
Can you try that query using `curl`? I think there's an error with marshaling the JSON, but the client is trying to parse a text line instead of showing the marshaling error.

```
$ curl -XPOST 'http://localhost:8086/query?db=<db>' --data-urlencode 'q=select "bytes_recv" from "net" where "host" = 'wowza1.ca.iptv' limit 10'
```

**Comment by jsternberg** [comment]
This might be related to #6846.

**Comment by jsternberg** [comment]
Fixed via #7293.
